### ISSUE TITLE
Central resource pack/shader pack folders

### ISSUE THREAD

**Issue by Aces-and-Jacks** [PrismLauncher/PrismLauncher#2056]
### Role

I play modded Minecraft

### Suggestion

I want there to be central folders for resource packs and shader packs similar to the mods folder

### Benefit

So I can have all my resource packs and shader packs I use a bunch in one central folder for both types

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

I tried searching if there was any similar issues "central resource packs" or "central shader packs" but the one issue that came up was a symlink resource pack folder which is separate. So if there's already an issue for this I missed it. 

Anyway, it'd also help for anyone who uses something similar or exact to ferium which I use for keeping mods I use for a lot of instances for different versions up to date (isn't supported by ferium yet but is planned to eventually). Since they could maybe specify a folder location on an external drive if they want to use it on other computers. It'd also probably apply to any reason that the central mod folder helps.

**Comment by josephrooks** [comment]
For a mod-based solution (or for a point of reference), [Shared Resources](https://modrinth.com/mod/shared-resources) on Modrinth does this for a few versions.

**Comment by Aces-and-Jacks** [comment]
Yeah I've heard about that mod. But it'd still be nice for the launcher to support central res/shader folders without mod support. And I wouldn't really wanna grow used to needing a mod for a feature that the launcher could implement similarly to the central mods folder, especially with potential compatibility bugs the mod could have with certain mods. Thanks for the suggestion tho

**Comment by Continous** [comment]
How is this different from #38

**Comment by Aces-and-Jacks** [comment]
> How is this different from #38

Because that's a symlink function. I want it to work how the central mods folder works

**Comment by Aces-and-Jacks** [comment]
Well I guess the shared packs is similar. But I just want to put a bunch of resource packs in one central folder without it showing up in a bunch of other instances.

If you want to be extra there can be a central folder and a shared folder. One for just storage the other for sharing across instances. But I don't think any of the developers would want that

**Comment by Continous** [comment]
Centralizing where all resource packs/shader packs are stored is just the inversion of issue #38. But I see your point. I think in generally, both issues are just expressions of a desire for Prism to properly manage resources across instances.

**Comment by Aces-and-Jacks** [comment]
So not a central folder. Hm, shame

**Comment by Continous** [comment]
> So not a central folder. Hm, shame

Can you perhaps clarify what you mean by this? What about this issue is unique from 38?

**Comment by Aces-and-Jacks** [comment]
> > So not a central folder. Hm, shame
> 
> Can you perhaps clarify what you mean by this? What about this issue is unique from 38?

What I want is to have everything grouped in one central folder, like how mods is. I can store a bunch of resource/shader packs in their own folders, and select where the folder is made. 

The other issue is about symlinking everything to a folder. You'd be able to select if certain instances would be able to access it sure. But what if I only want to use some picks and not everything. And since it can be used with a bunch of other instances that'd create conflicts.
